packages/fcl/src/current-user/index.js

```diff
@@ -78,7 +78,7 @@ function notExpired(user) {
   )
 }
 
-async function authenticate() {
+async function authenticate(opts = {redir: false}) {
```

Comment:
Can we maybe get a warning about this option if they use it. Something like "hey you are manually enabling a very experimental feature that is not yet standard, use at your own risk, you can disable this warning by setting fcl.warning.suppress.redir to true in your config"

Reply:
Added warning, suppressed if `fcl.warning.suppress.redir`: `true`